ipn/localapi/cert.go

```diff
@@ -23,7 +23,7 @@ func (h *Handler) serveCert(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "internal handler config wired wrong", 500)
 		return
 	}
-	pair, err := h.b.GetCertPEM(r.Context(), domain)
+	pair, err := h.b.GetCertPEM(r.Context(), domain, true)
```

Comment:
should users opt-in to this with a CLI flag? Making it randomly slow sometimes seems like a regression for people who were using it the old way and assuming it always served from cache.

We should at least leave have a flag to let you still have the old behavior.

But at that point, why not a time.Duration instead of a bool? Let you specify the validity window you're cool with cache-wise. Because the author knows their cron schedule but we/ARI don't know.

Reply:
I like the idea of a duration, it plays nicely with a flag too.